py/selenium/webdriver/remote/webdriver.py

```diff
@@ -1222,3 +1222,52 @@ def delete_downloadable_files(self) -> None:
             raise WebDriverException("You must enable downloads in order to work with downloadable files.")
 
         self.execute(Command.DELETE_DOWNLOADABLE_FILES)
+
+    # Federated Credential Management (FedCM)
+    def get_fedcm_title(self) -> str:
+        """Gets the title of the dialog."""
+        return self.execute(Command.GET_FEDCM_TITLE)["value"].get("title")
+
+    def get_fedcm_subtitle(self) -> Optional[str]:
+        """Gets the subtitle of the dialog."""
+        return self.execute(Command.GET_FEDCM_TITLE)["value"].get("subtitle")
+
+    def get_fedcm_dialog_type(self):
+        """Gets the type of the dialog currently being shown."""
+        return self.execute(Command.GET_FEDCM_DIALOG_TYPE).get("value")
+
+    def get_fedcm_account_list(self):
+        """Gets the list of accounts shown in the dialog."""
+        return self.execute(Command.GET_FEDCM_ACCOUNT_LIST).get("value")
+
+    def select_fedcm_account(self, index: int) -> None:
+        """Selects an account from the dialog by index."""
+        self.execute(Command.SELECT_FEDCM_ACCOUNT, {"accountIndex": index})
+
+    def click_fedcm_dialog_button(self) -> None:
```

Comment:
If this is about clicking continue then maybe we should have this as

```suggestion
    def continue(self) -> None:
```

Reply:
Changing to `click_continue` as `continue` is a reserved keyword in python.

Reply:
Let's move it to be like alerts then and have it as `accept`

Reply:
I didn't got the alerts reference, can you please elaborate a bit more?